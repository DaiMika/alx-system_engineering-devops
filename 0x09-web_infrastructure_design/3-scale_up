https://github.com/DaiMika/alx-system_engineering-devops/blob/master/0x09-web_infrastructure_design/3-scale_up.draft

https://github.com/DaiMika/alx-system_engineering-devops/blob/master/0x09-web_infrastructure_design/snap/3-scale_up.snap
