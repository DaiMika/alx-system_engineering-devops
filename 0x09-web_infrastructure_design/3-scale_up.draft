https://docs.google.com/document/d/1QTUcnECox47Nfubmv-RFrmaovMoh9vc-Q18BM1zeaVo/edit?usp=drivesdk
