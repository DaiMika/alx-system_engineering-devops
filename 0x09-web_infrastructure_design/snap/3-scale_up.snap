https://docs.google.com/document/d/1DCuYC-O346msUO1OJSaxRBxYEkIjNXlNLMqwLoLoiRo/edit?usp=drivesdk
